package.lisp

```diff
@@ -19,9 +19,20 @@
    #:*parse-json-booleans-as-symbols*
    #:*parse-json-null-as-keyword*
 
+   
+   #:*allow-nan*     
+   #:*yason-float-parser* 
```

Comment:
Please drop the `yason-` prefix - the package should make clear where the symbols are from.